ext/openssl/openssl.c

```diff
@@ -1845,7 +1845,7 @@ zend_string* php_openssl_x509_fingerprint(X509 *peer, const char *method, zend_b
 	zend_string *ret;
 
 	if (!(mdtype = EVP_get_digestbyname(method))) {
-		php_error_docref(NULL, E_WARNING, "Unknown signature algorithm");
+		php_error_docref(NULL, E_WARNING, "Unknown hashing algorithm");
```

Comment:
Shouldn't this also use "digest"?

Reply:
According to https://www.php.net/manual/es/function.openssl-x509-fingerprint.php it has a `hash_algorithm` name and is: `The digest method or hash algorithm to use, e.g. "sha256", one of openssl_get_md_methods().`

Though, you must be right since `openssl_get_md_methods()` returns the available digest algorithms.

Reply:
I agree that it would be better to use digest to be more consistent and the documentation should be updated.